pkg/universal-cnf/vppagent/operations.go

```diff
@@ -111,7 +111,10 @@ func SendVppConfigToVppAgent(vppconfig *vpp.ConfigData, update bool) error {
 		_, err = client.Delete(ctx, &configurator.DeleteRequest{
 			Delete: dataChange,
 		})
+		if err != nil {
+			return err
+		}
 	}
 
-	return err
+	return nil
```

Comment:
This will actually make updates not return any errors. 

* if `update==true` =>
  * calling `client.Update` returns error =>
  * bare log of error without any info =>
    (would be good to add `Update failed: %v` at least)
  * calling `client.Delete` =>
  * no logging on error =>
* then returns `nil`

Either add `if err != nil return err` to the update case or do it after the entire `if ... else ...`.

If this was actually your intention please explain the reasoning behind it.

Reply:
It actually wasn't my intention, I think accidentally modified the last return statement without noticing it. Thank you for pointing it out